cpp/daal/src/algorithms/dtrees/gbt/oneapi/gbt_feature_type_helper_oneapi.i

```diff
@@ -89,6 +89,7 @@ template <typename algorithmFPType>
 IndexedFeaturesOneAPI<algorithmFPType>::~IndexedFeaturesOneAPI()
```

Comment:
factory.build(ExecutionTargetIds::device, cachekey.c_str(), gbt_common_kernels, build_options.c_str());  - status check?
__buildProgram should return Status and it should be checked everywhere

Reply:
Fixed.